src/jit/hwintrinsicArm64.cpp

```diff
@@ -109,15 +109,17 @@ NamedIntrinsic Compiler::lookupHWIntrinsic(const char* className, const char* me
     NamedIntrinsic result = NI_Illegal;
     if (isa != InstructionSet_NONE)
     {
+        JITDUMP("HW Intrinsic %s.%s: ", className, methodName);
```

Comment:
Might be nice to centralize this logging in `Compiler::lookupNamedIntrinsic` in `importer.cpp`? That would allow all named intrinsics to have JITDUMP info and a fallback for not supported/not recognized.

Reply:
Currently that won't really get us what we'd really like because the VM won't even surface the intrinsics for other platforms. I'll add a note to #20379 that we should also unify the `JITDUMP` output for intrinsic recognition.